### DISCUSSION TITLE
DSCR Describes what?

### DISCUSSION THREAD

**Discussion by mother10** [FamilySearch/GEDCOM#341] [Q&A] [answered]
I read the Gedcom 5.5.1 and the latest Gedcom 7.
Gedcom 5.5.1 says:

An unstructured list of the attributes that describe the physical characteristics of a person, place, or object. Commas separate each attribute.

So to me it looks as if you can describe a persons characteristics, but also those of a place or a thing. (a book perhaps?)
Searched everywhere, but could not find any examples of the last 2, just for a person.

Gedcom 7 says:

The physical characteristics of a person.

So where are the place and the thing? No longer present in Gedcom7?

I would like to know what it can be used for with some examples please.
Thanks a lot.
Mother10

**Comment by tychonievich** [comment, marked as answer]
We changed its definition in 7.0 to match its use in 5.5.1, which included two changes:

The tag was introduced in 5.0 as a substructure of `INDI`, and only in that context, which was also observed in all subsequent versions. Hence, saying it could describe a place or object was misleading as it could only appear as a descriptor of a person, so we removed that part of its definition.

The comma-separated aspect appeared to be widely ignored in practice, with payloads like "large ears, eyes, and nose" using commas grammatically and not as separators. Separate descriptions were more often encoded with multiple `DSCR`. When multiple descriptors where in one `DSCR` with commas in between, the payload could be read as a single piece of text without losing meaning. Hence, we removed the comma-separated part of its definition.

**Reply by mother10** [reply]
Thanks very much for the explanation !
